ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/Command.java

```diff
@@ -24,20 +24,24 @@
 
 @JsonSubTypes({})
 public class Command {
+  public static final long VALID_CHECKSUM = -1L;
```

Comment:
I put a lot of time thinking about the naming for this and I couldn't come up with something that I liked. The terminology that I settled on was:
* `offset`: the actual offset on the kafka topic
* `checksum`: the offset that it was validated against

I tried not to conflate the two so that it's obvious when we check checksum against offset what is happening. Let me know if you have any better terminology or if you disagree.

I'm happy changing it to `NO_CHECKSUM`

Reply:
Makes sense. I'm fine with it as is.

Reply:
I have no issue with the `offset` vs `checksum`, but I'm with @rodesai on the `NO_CHECKSUM`, given that `-1` isn't a valid checksum or offset ;)